common/config/rush/pnpm-lock.yaml

```diff
@@ -674,10 +674,10 @@ importers:
       '@itwin/eslint-plugin': link:../../tools/eslint-plugin
       '@itwin/presentation-common': link:../../presentation/common
       '@types/node': 14.14.31
-      electron: 14.2.0
+      electron: 14.1.1
```

Comment:
You appear to have downgraded electron and typescript. That's unintentional, right?

Reply:
Yeah that's unintentional, not sure why that happened. That should be fixed with the latest commit.